content/product/product_management/index.md

```diff
@@ -53,6 +53,35 @@ Part of the product data is a (internal-only) list of [what's supported](https:/
 
 Launch tier levels (L1, L2, and L3) are also an important term to be aware of, and the definitions as well as process/source of truth are documented on the [marketing launch page](../../marketing/product-marketing/marketing_launch_tiers.md).
 
+## Collaboration
+
+Product Management tends to collaborate with a lot of different groups. Beyond our shared [work as a team value](../../company/values.md#work-as-a-team), there are some teams where collaboration is especially important or can be unclear so we have documented high level practices around how we work together.
+
+### Product Management & Marketing
+
+Product Management and Marketing must work closely together to successfully create and launch products. Because of this, it's important to be clear on what the shared and unique responsibilities are within the working relationship.
+
+TODO add link to source of this info
+
+| Product Marketing is DRI                                                                                 | Product Management is DRI                                                        |
+| -------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| Customer and market-facing                                                                               | Development and product-facing                                                   |
+| Defining the roadmap                                                                                     | Providing important market context to the roadmap                                |
```

Comment:
I'm not sure that this one is accurate of the stat we are at today. I think Product is currently owning defining the roadmap.

Reply:
Oops, you're right! I swapped the two columns.

```suggestion
| Providing important market context to the roadmap                                                                                     | Defining the roadmap                                |
```